File: years/2015/elixir/test/aoc/day_05_test.exs
defmodule AoC.Day05.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day05

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day05.part_1(["../data/05.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day05.part_2(["../data/05.txt"]) == 1
    end
  end
end

File: lang/clojure/skeleton/test/day-05-test.clj
(ns day-05-test
  (:require
   [clojure.test :refer :all]
   [day-05 :refer :all]))

(deftest day-05-test
  (testing "sample data"
    (is (= 0 (part1 "../data/05-sample.txt")))
    (is (= 0 (part2 "../data/05-sample.txt"))))
  (testing "puzzle data"
    (is (= 0 (part1 "../data/05.txt")))
    (is (= 0 (part2 "../data/05.txt")))))
